### PR TITLE
[NUI] add support for hyphen + mixed line wrap

### PR DESCRIPTION
**Pull request by abdullehghujeh**
### Description of Change ###
extend the functionality of line wrapping by adding hyphenation & mixed modes
depends on : 
   https://review.tizen.org/gerrit/c/platform/core/uifw/dali-toolkit/+/254279/
   https://review.tizen.org/gerrit/c/platform/core/uifw/dali-adaptor/+/254278

### API Changes ###
 - ACR: None

Changed:
 - add Hyphenation & Mixed mode to line wrap enum

### CODE REVIEW

File: src/Tizen.NUI/src/public/Common/NUIConstants.cs [hunk 1]
```diff
@@ -807,7 +807,20 @@ public enum LineWrapMode
...
         /// <since_tizen> 4 </since_tizen>
-        Character
+        Character,
+
+        /// <summary>
+        /// Hyphenation mode will move part of the word (at possible hyphen locations)
+        /// to the next line and draw a hyphen at the end of the line.
+        /// </summary>
+        /// <since_tizen> ? </since_tizen>
+        Hyphenation,
```
Comment (huiyueun) [inline src/Tizen.NUI/src/public/Common/NUIConstants.cs:817]: If you want to add Public API, you should do ACR process. If not, add it as a hidden API. ,
you can add tag '[EditorBrowsable(EditorBrowsableState.Never)]' to make hidden API.

**Comment by dongsug-song** [comment]
@abdullehghujeh 
Hello, 
could you write the reply and click the button on "Resolve Conversation" after you fix the patch?
and could you request review again by clicking the **recycle icon** which attached in front of reviewer?
thank you.

File: src/Tizen.NUI/src/public/Common/NUIConstants.cs (hunk 1 above)
Comment (abdullehghujeh) [reply src/Tizen.NUI/src/public/Common/NUIConstants.cs:817]: I have added the the hidden API tags

**Comment by dongsug-song** [comment]
@abdullehghujeh 
Hello, 
I think the dali patches that you have mentioned in description should be merged prior to this patch is reviewed.
otherwise it seems that when the newly added property is used, there will be a malfunction.

now I put the \*\***NO MERGE**\*\* tag on thie PR.
you can remove \*\***NO MERGE**\*\* tag for yourself when this PR is fixed and ready to be merged.
(\*\***NO MERGE**\*\* tag has no restriction to submit PR or to editing this patch but just for notification and marking.
thank you.

**Comment by wonrst** [comment]
@abdullahghujah 
Close this PR because, the same patch has been merged into the DevelNUI branch.
https://github.com/Samsung/TizenFX/pull/3019

**Pull request closed without merging**